lib/vtls/schannel.h

```diff
@@ -61,16 +71,79 @@ CURLcode Curl_verify_certificate(struct Curl_easy *data,
 
 #ifdef __MINGW32__
 #include <_mingw.h>
-#ifdef __MINGW64_VERSION_MAJOR
+#if defined(__MINGW64_VERSION_MAJOR) && !defined(DISABLE_SCHANNEL_MANUAL_VERIFY)
```

Comment:
I think the two new flags/options `DISABLE_SCHANNEL_MANUAL_VERIFY` and `DISABLE_SCHANNEL_CLIENT_CERT` should be done in a separate PR as this is unrelated to TLS 1.3.

> (SChannel already handles verification internally)

Also this is not true for all versions of Windows.

Reply:
> Also this is not true for all versions of Windows.

Really? From my research (again, documentation is spotty on MSDN), auto-verification has been supported since at least Windows 2000.

Either way, I can make a separate PR that adds those 2 options. Or better yet, adds the inverse of `DISABLE_SCHANNEL_MANUAL_VERIFY` (`ENABLE_SCHANNEL_MANUAL_VERIFY`) so that by-default end-users will be smaller binaries.

Would that ease the merging of this PR? If so, I'll do that.

Reply:
See the commit originally introducing it: https://github.com/curl/curl/commit/1e4c57fa6471c0010f66d82a83f190a07fa37309

Reply:
Ah, so it was added for Windows CE. So, it makes sense to just enable manual verification for Windows CE only. So, do away with `DISABLE_SCHANNEL_MANUAL_VERIFY` and just use `_WIN32_WCE` (which looks like the original behavior implemented).

Reply:
OK, I've removed `DISABLE_SCHANNEL_MANUAL_VERIFY` and used `_WIN32_WCE` as was originally written.